env/CMakeLists.txt

```diff
@@ -67,4 +68,19 @@ ExternalProject_Add(
     DEPENDS python-venv
 )
 
+ExternalProject_Add(
+    onnx-mlir
+    PREFIX ${TTMLIR_TOOLCHAIN_DIR}
+    CMAKE_GENERATOR Ninja
+    CMAKE_ARGS
+    -DCMAKE_CXX_COMPILER=/usr/bin/c++
+    -DPython3_ROOT_DIR=$pythonLocation
+    -DMLIR_DIR=${TTMLIR_TOOLCHAIN_DIR}/llvm-project/build/lib/cmake/mlir
+    -DCMAKE_INSTALL_PREFIX=${TTMLIR_TOOLCHAIN_DIR}
+    GIT_REPOSITORY https://github.com/onnx/onnx-mlir/
+    GIT_TAG ${ONNX_MLIR_VERSION}
+    GIT_SUBMODULES_RECURSE true
+    DEPENDS python-venv
+)
```

Comment:
This `ExternalProject_Add` should be moved to the `third_party` subdirectory.  I think we should:

```
git mv third_party/CMakeLists.txt third_party/tt-metal.cmake
```

You should create `third_party/onnx-mlir.cmake` along with a new `CMakeLists.txt`:
```
include(tt-metal)
if (TTMLIR_ENABLE_ONNX_MLIR)
include(onnx-mlir)
endif
```

Reply:
Ok got it, let me update with the changes.